WalletWasabi.Tests/PasswordTests.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+using WalletWasabi.Helpers;
+using Xunit;
+
+namespace WalletWasabi.Tests
+{
+	public class PasswordTests
+	{
+		[Fact]
+		public void PasswordClipboardCutTest()
```

Comment:
I think this UT is not enough. The main method to test seems to be `PasswordHelper::GetMasterExtKey`.

Reply:
True, adding more tests.